packages/mobile/src/components/overflow-menu-drawer/TrackOverflowMenuDrawer.tsx

```diff
@@ -176,13 +180,10 @@ const TrackOverflowMenuDrawer = ({ render }: Props) => {
       navigation?.push('EditTrack', { id })
     },
     [OverflowAction.RELEASE_NOW]: () => {
-      dispatch(
-        setVisibility({
-          drawer: 'PublishContentDrawer',
-          visible: true,
-          data: { contentId: id, contentType: 'track' }
-        })
-      )
+      openPublishConfirmation({
+        contentType: 'playlist',
+        confirmCallback: dispatch(trackPageActions.makeTrackPublic(id))
```

Comment:
confirm callback should be wrapped in a function

Reply:
great catch on these